cockroachcloud/ccloud-get-started.md

```diff
@@ -317,6 +326,35 @@ warning: server version older than client! proceed with caution; some features m
 user@free-tier7.gcp-us-central1.crdb.io:26257/defaultdb>
 ~~~
 
+### Connect to your cluster using SSO
+
+Use the `--sso` flag to connect to your cluster using [single sign-on (SSO) authentication](cloud-sso-sql.html). 
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql --sso dim-dog
+~~~
+
+This will open a browser window on the local machine where you will login to your organization if you are not already authenticated.
+
+If you are running `ccloud` on a remote machine, use the `ccloud cluster sql -sso --no-redirect` flag, or type "n" when prompted by `ccloud` to redirect to your browser. `ccloud` will output a URL that you must copy and paste in your local machine's browser in order to authenticate. After authentication, paste in the authentication code you received in the remote terminal to complete the login process.
```

Comment:
* `--sso` not `-sso`
* we can remove the part about typing "n", we removed that in favor of the `--no-redirect` flag

Reply:
Done.